### ISSUE TITLE
Changing the shell directory didn’t work as expected

### ISSUE THREAD

**Issue by blues-man** [openshift-instruqt/instruqt#13]
The CLI starts with root users on `/`, it should start to `/root` as `$HOME`

Either using `cd /root` in `.bashrc` or `.bash_profile` didn't worked, neither `config.yml` since this is a nested VM.

**Comment by amochtar** [comment]
Not sure how it being a nested VM is related, but it seems `$HOME` is not set somehow:

```
[root@crc-dzk9v-master-0 /]# cd
bash: cd: HOME not set
```

When I add these lines to `/root/.bashrc`, I do start in `/root`:

```
export HOME=/root
cd $HOME
```

**Comment by blues-man** [comment]
Hi @amochtar , the issue is that when you start the track, the $HOME directory is not set. If you do `su - root` then it is set.

I tried to append in the setup script the export + cd in the bash_profile, but the terminal starts always in / and $HOME is not set

**Comment by amochtar** [comment]
Yes, but `$HOME` is not something we explicitly set or unset. The OS should set that, as it's part of the [POSIX standard](https://pubs.opengroup.org/onlinepubs/009695399/basedefs/xbd_chap08.html). Could it be some glitch in the crc image?

**Comment by yuriolisa** [comment]
Hi, @blues-man, 

I just found an issue on scenario 2 of Operator SDK with Go, in which the root cause is the $HOME is not set. 

`go: downloading golang.org/x/mod v0.2.0
build cache is required, but could not be located: GOCACHE is not defined and neither $XDG_CACHE_HOME nor $HOME are defined
make: *** [Makefile:126: controller-gen] Error 1
Error: failed to create API: unable to run post-scaffold tasks of "base.go.kubebuilder.io/v3": exit status 2`

Could you please let me know if I should open a new issue or keep it as a comment here?

**Comment by blues-man** [comment]
That happens when the context is not loaded, it is not clear why CoreOS (Nested VM) starts in this way, and we cannot inject the setting from the Host VM.

As a workaround I added in that track the command `su - root` but now we are also working on sidecar container based tracks like this:
https://github.com/openshift-instruqt/instruqt/blob/master/operatorframework/test/operatorframework-go-operator-podset-sidecar/config.yml

**Comment by ryanj** [comment]
This issue is resolved in the 4.16 image.  We will need to update all tracks to use the newer image
